models/generics/contact_information.py

```diff
@@ -60,10 +60,14 @@ class Meta:
 
         verbose_name_plural = 'contact information'
         constraints=[
-            models.UniqueConstraint(fields=['entity_content_type', 'primary_phone_number'], name='unique_primary_number'),
-            models.UniqueConstraint(fields=['entity_content_type', 'secondary_phone_number'], name='unique_secondary_number'),
-            models.UniqueConstraint(fields=['entity_content_type', 'email_address'], name='unique_email_address'),
-            models.UniqueConstraint(fields=['entity_content_type', 'institute_webmail_address'], name='unique_institute_webmail_address'),
+            models.UniqueConstraint(fields=['entity_content_type', 
+            'primary_phone_number'], name='unique_primary_number'),
```

Comment:
This does look overly long but it's the convention followed across all models in the codebase.
```suggestion
            models.UniqueConstraint(
                fields=[
                    'entity_content_type', 
                    'primary_phone_number',
                ], 
                name='unique_primary_number',
            ),
```

Reply:
Note the trailing commas _everywhere_.